sample-android-compose/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+plugins {
+  id("com.android.application")
+  id("org.jetbrains.kotlin.android")
+  // Used for Compose Destinations(https://github.com/raamcosta/compose-destinations)
+  id ("com.google.devtools.ksp") version "$kotlin_version-1.0.6"
```

Comment:
can we also avoid hardcoding the `1.0.6` version?

We could have the `ksp` as an entire variable.